### DISCUSSION TITLE
TILE based placement

### DISCUSSION THREAD

**Discussion by salmen20** [The-OpenROAD-Project/OpenROAD-flow-scripts#2632] [General]
Hello everyone ,
I currently have a tile based design and actually any other design is just a replication of it what i wanna do is to manually place those tile blocks beside each other as we can numerate them on a matrix form [[1,2,3],[4,5,6],[7,8,9]] the other design woul be just [[1,2,2,3],[4,5,5,6],[7,8,8,9]] is there a way to control the tile placement to be the same as the exmpla given and not random as i wanna the block 1 to be at the top left and same for any other tile number . 
if theres any suggestion on how should i approach this problem .
Thanks.

**Comment by maliberty** [comment]
You could use manual macro placement or see if the automatic macro placer will give you what you want.  See https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/tree/master/flow/designs/asap7/mock-array as an example.

**Reply by salmen20** [reply]
i can run the whole flow for each number alone , is there a way where i could just import them as blocks ? also for the macro placer how can i find the macro name of my tiles ?

**Reply by maliberty** [reply]
mock-array shows build a block and then arraying it.
